Source/Core/VideoBackends/Software/TransformUnit.cpp

```diff
@@ -137,6 +137,15 @@ static void TransformTexCoordRegular(const TexMtxInfo& texinfo, int coordNum,
   }
   }
 
+  // Convert NaNs to 1 - needed to fix eyelids in Shadow the Hedgehog during cutscenes
+  // See https://bugs.dolphin-emu.org/issues/11458
+  if (src.x != src.x)
```

Comment:
`std::isnan` would be a little more self-documenting here

Reply:
Apparently [glsl](https://www.khronos.org/registry/OpenGL-Refpages/gl4/html/isnan.xhtml) and [hlsl](https://docs.microsoft.com/en-us/windows/win32/direct3dhlsl/dx-graphics-hlsl-isnan) both have `isnan` too.  I'll switch to them.